src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptorTest.java

```diff
@@ -499,6 +499,13 @@ public Object invokeMethod(String name, Object args) {
         return x + 2;
     }
 
+    @Issue("JENKINS-32211")
+    @Test public void tokenize() throws Exception {
+        assertEvaluate(new GenericWhitelist(), 3, "'foo bar baz'.tokenize().size()");
+        assertEvaluate(new GenericWhitelist(), 3, "'foo bar baz'.tokenize(' ').size()");
+        assertEvaluate(new GenericWhitelist(), 3, "'foo bar baz'.tokenize('ba').size()");
+    }
```

Comment:
Do not really need test coverage for simple whitelist additions---only in cases where we think something funny is going on in the code.